### ISSUE TITLE
Issue with viewing Module Positions on Front-End Joomla! 4

### ISSUE THREAD

**Issue by 200MPHMEDIA** [joomla/joomla-cms#24001]
If you view the Module Positions in the new Template, some of them are cut off and not visible.

Try the ?tp=1 view after enabling the plugin.

Anyone considering putting those display boxes on a higher z-index so they can be seen in full and not cut off, either fully or partially.

Those with some "juice" with the developers: please pass this along to them.

Thanks.

**Comment by ciar4n** [comment]
This issue is likely to be resolved once https://github.com/joomla/joomla-cms/pull/22272 is merged into 4.0-dev.

**Comment by 200MPHMEDIA** [comment]
Thank you.  Will look for the update.

**Comment by ghost** [comment]
@200MPHMEDIA please close the Issue as its resolved.
